nms/app/packages/magmalte/alerts/lteAlerts.js

```diff
@@ -56,6 +56,23 @@ export default function getLteAlerts(
         description: 'Alerts when we have unexpected service restarts',
       },
     },
+    'Service Crashlooping Alert': {
+      alert: 'Service Crashlooping Alert',
+      expr: `increase(unexpected_service_restarts{networkID=~"${networkID}"}[5m]) > 5`,
+      labels: {severity: 'critical'},
+      annotations: {
+        description: 'Alerts when we have services crashlooping',
```

Comment:
there's no remediation available for this?

Reply:
No. This covers all services. We don't have sufficient insight to provide when any service restarts rapidly